src/program/lwaftr/run/run.lua

```diff
@@ -39,19 +39,18 @@ function parse_args(args)
    if #args == 0 then show_usage(1) end
    local conf_file, v4, v6
    local ring_buffer_size
-   local opts = { verbosity = 0, ingress_drop_monitor = 'flush' }
+   local opts = {
+      verbosity = 0, ingress_drop_monitor = 'flush', csv_file = 'bench.csv' }
    local handlers = {}
    local cpu
    function handlers.v () opts.verbosity = opts.verbosity + 1 end
    function handlers.i () opts.virtio_net = true end
    function handlers.D (arg)
       opts.duration = assert(tonumber(arg), "duration must be a number")
+      assert(opts.duration >= 0, "duration can't be negative")
    end
    function handlers.c(arg)
       conf_file = arg
-      if not arg then
-         fatal("Argument '--conf' was not set")
-      end
```

Comment:
These checks are redundant: specifying the option without the mandatory parameter is already caught by the library, because of the colon marker in options declaration.